CMakeLists.txt

```diff
@@ -144,6 +144,7 @@ option(NGRAPH_ARTIFACTS_DIR "Where would nGraph be installed after build" FALSE)
 option(UNIT_TEST_ENABLE "Control the building of unit tests" FALSE)
 option(UNIT_TEST_TF_CC_DIR "Location where TensorFlow CC library is located" FALSE)
 option(NGRAPH_DISTRIBUTED_ENABLE "Add distributed mode to the CPU backend" FALSE)
+option(NGRAPH_BRIDGE_STATIC_LIB_ENABLE "Enable build nGraph static library" TRUE)
```

Comment:
Once it is tested and reviewed I will set this to `FALSE`

Reply:
Is this by default `TRUE` or `FALSE`

Reply:
I can keep it TRUE or just generate both .a and .so by default without adding any option. @avijit-nervana let me know which version you want. Right now we set this TRUE and link gtest and cpp with static lib.

Reply:
We need to keep both static and DSO versions.